code-challenges.js

```diff
@@ -30,40 +43,97 @@ const temperature2 = 350
 
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
+console.log(boilingPointTemp(temperature1))
+console.log(boilingPointTemp(temperature2))
+console.log(boilingPointTemp(temperature3))
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// combine 2 arrays......concat? also well need to return the combined length
+// varible combining arrays and then another varible strictly for combining the arrays
+// return the the combined arrays 
+// create another varible that I can console.log the the combined arrays
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+const combineArray = (arr1, arr2) => {
+  const combineRuns = arr1.concat(arr2)
+    return combineRuns.length
+}
+
+const totalLength = combineArray(padres1984WorldSeriesRuns, padres1998WorldSeriesRuns)
+console.log(totalLength)
+
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
-
+// create a variable to reverse the string 
+// split the string in to an array 
+// reverse the array of the letter and numbers
+// join the reveresed outcome 
 const currentCohort = "Echo 2023"
+// creating const for reverse str
+const reString = (str) => {
+  //splitting the array
+  const strArray = str.split("")
+  //reversing the array
+  const reArray = strArray.reverse()
+  // joining the array
+  const reString = reArray.join("")
+  return reString
+}
+const reCohort = reString(currentCohort)
+
+console.log(reCohort)
 // Expected output: "3202 ohcE"
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
+// return the last index value from an array
+// need to output the the giveValues index space with the given number
+// index starts with zero
 
 const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 
+const lastIndexValue = (arr, value) => {
+  return arr.lastIndexOf(value)
+}
+// lastIndexOf returns the index of the last number
```

Comment:
Good logic, make sure your pseudo is clear. `lastIndexOf` returns the index of the last occurrence of the argument passed to it.